eng/Build.ps1

```diff
@@ -212,7 +211,7 @@ function UpdatePath() {
 }
 
 function VerifyAssemblyVersions() {
-    $fsiPath = Join-Path $ArtifactsDir "bin\fsi\Proto\net472\publish\fsi.exe"
+    $fsiPath = Join-Path $ArtifactsDir "bin\fsi\Proto\net472\fsi.exe"
```

Comment:
Publishing ensures the right dependencies are in the folder, hence it can be copied accurately to the "Bootstrap" directory

Reply:
@dsyme, the build has to do that anyway, otherwise f5 debugging and build test doesn't work.